pkg/tfbridge/property_path.go

```diff
@@ -112,22 +127,177 @@ func propertyValueTriggersReplacement(
 	return replacement
 }
 
-// pathContainsComputed returns true if the schema contains a Computed property at a path prefixed by path.
-func pathContainsComputed(
-	path propertyPath, rootTFSchema shim.SchemaMap, rootPulumiSchema map[string]*info.Schema,
+// isPlanCompatibleWithInputs returns true if all values in the walkedValue are also in the comparedValue,
+// bar any computed properties.
+func isPlanCompatibleWithInputs(
```

Comment:
This is really not very easy to verify. What is this function trying to accomplish? Equality up to computed?